src/Provider/IslandoraServiceProvider.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Islandora\Crayfish\Commons\Provider;
+
+use Islandora\Crayfish\Commons\CmdExecuteService;
+use Islandora\Crayfish\Commons\FedoraResourceConverter;
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Silex\Provider\DoctrineServiceProvider;
+use Silex\Provider\MonologServiceProvider;
+use Silex\Provider\ServiceControllerServiceProvider;
```

Comment:
@jonathangreen that is only needed (at app level) to allow Controllers (route controllers, like the code that reacts to GET or POST) to be defined as services, its a way of overriding the normal controller resolver to allow classes to be used , stuff like this
$app->get('/ldp', "posts.controller:givememyJson");

I don't see that here. The documentation there is a bit misleading but if you look further down it expands to the controller notion.